src/runtime_config/default/actions.js

```diff
@@ -672,6 +711,9 @@ const actions = new Map([
         const codeId = ccf.strToBuf(args.code_id);
         const ALLOWED = ccf.jsonCompatibleToBuf("AllowedToJoin");
         ccf.kv["public:ccf.gov.nodes.code_ids"].set(codeId, ALLOWED);
+
+        // Adding a new allowed code ID changes the semantics of any other open proposals, so invalidate them to avoid confusion or malicious vote modification
+        invalidateOtherOpenProposals(proposalId);
```

Comment:
Can we also test for this in `code_update.py`?

Reply:
Yes, I'll add this.